app/frontend/src/components/views/project-view/ProjectView.tsx

```diff
@@ -22,51 +22,38 @@ interface Props { }
  */
 const ProjectView: React.FC<Props> = () => {
   const params = useParams();
-  const [tasksList, setTasksList] = useState<Task[]>([]);
   const projects = useStore($projects);
+  const tasksList = useStore($tasks);
   const currentProject = projects.find((project) => params.id === project._id);
   const title = currentProject?.title || 'All projects';
 
+
   useEffect(() => {
-    (async function fetchTasks() {
-      try {
-        const { tasks } = await getTasks(params.id ? { projectId: params.id } : {});
-
-        setTasksList(tasks);
-      } catch (e) {
-        console.error(e);
-      }
-    })();
+    getTasksEffectFx(params.id ? { projectId: params.id } : {});
   }, [ params.id ]);
 
   const createNewTask = async (value: string): Promise<void> => {
-    try {
-      const taskContent = {
-        blocks: [
-          {
-            type: 'header',
-            data: {
-              text: value,
-              level: 1,
-            },
+    const taskContent = {
```

Comment:
this logic should be on the backend side

Reply:
i dont think that backend should know anything about editorjs data format. From backend perspective task content is just a text